git-info-generator.mjs

```diff
@@ -0,0 +1,22 @@
+import { execSync } from 'child_process';
+import { writeFileSync } from 'fs';
+import shell from 'shelljs';
+
+if (!shell.which('git')) {
```

Comment:
No need to pull in the shelljs dependency for this. Something like this should work:
```js
try {
  execSync('git --version');
} catch (error) {
  console.error(error.message);
  throw new Error(`git is missing, please install git and retry`);
}
```

Reply:
do you think It would work on windows?